app/controllers/image-set.js

```diff
@@ -32,6 +32,18 @@ export default Ember.Controller.extend({
 
         controller.transitionToRoute('image-set', imageSet);
       });
+    },
+
+    requestCv: function() {
+      var imageSet = this.get('model'),
+          currentUser = this.get('toriiSession.currentUser');
+
+      var cvRequest = this.store.createRecord('cvRequest', {
+        imageSet: imageSet,
+        uploadingOrganization: currentUser.get('organization')
+      });
+
+      cvRequest.save();
```

Comment:
I would like to see a little more "in-progress" feedback shown in the UI to the user. I've noticed that in general there is no UI feedback when ajax requests are in-flight. Having some text and/or a spinner that says "saving" or "loading" or, in the case of this action, perhaps a "making CV Request", would be very helpful for high-latency users. Otherwise I think we risk zalgo bugs where a user clicks "request cv" and then clicks away to another page or closes the window while the request is still in flight.

You could show a "loading"/"saving" message in the UI by using a flag here:
```
component.set('savingCVRequest', true);
component.set('errorSavingCVRequest', false);

cvRequest.save().then(function(){
  // no-op
}).catch(function(e){
  component.set('errorSavingCVRequest',true);
  // do something with error message?
}).finally(function(){
  component.set('savingCVRequest', false);
});
```

In other cases you could just check the model's isLoading or isSaving property in the template:
```
{{#if model.isSaving}}
Saving model...
{{else}}
<button>Click to save</button>
{{/if}}
```